recipes-core/initrdscripts/init-restore-mode_1.0.bb

```diff
@@ -12,17 +12,17 @@ SRC_URI = "\
 	file://ni_provisioning.answers.default \
 "
 
-SRC_URI_append_xilinx-zynqhf = "\
+SRC_URI:append:xilinx-zynqhf = "\
 	file://disk_config_xilinx-zynqhf \
 "
```

Comment:
Do we have a tracking item to remove this?

Reply:
Yep! https://dev.azure.com/ni/DevCentral/_workitems/edit/2112096